egs2/aphasiabank/asr1/local/spk_info.txt

```diff
@@ -0,0 +1,744 @@
+ACWT01a	63.9	moderate	broca
```

Comment:
Similar to the last comment, we may consider to hold the info in a separate repo.